docs/user/getting_started.rst

```diff
@@ -16,21 +16,45 @@ We recommend installing python via the anaconda distribution:
 
 Anaconda comes with many useful python libraries pre-installed and makes it easy to run CUQIpy code
 via the jupyter notebook app. In addition, CUQIpy plugins often require 3rd party libraries that can most
-easily be installed through anaconda.
+easily be installed through anaconda. |Miniconda| is a lightweight alternative to Anaconda if storage is a concern.
+
+.. |Miniconda| raw:: html
+
+   <a href="https://docs.anaconda.com/miniconda/miniconda-install/" target="_blank">Miniconda</a>
+
 
 .. _install:
 
 Installation
 ------------
 
-Installing CUQIpy is easy. Open your terminal (Linux and Mac) or Anaconda Prompt (Windows) and install it using pip:
+Before installing CUQIpy, We highly recommend creating a new Python environment using conda. This will ensure that CUQIpy's dependencies do not interfere with other Python packages you may have installed. Here we choose `cuqipy` as the environment's name and also install pip in it.
```

Comment:
```suggestion
Before installing CUQIpy, We highly recommend creating a new Python environment using conda. This will ensure that CUQIpy's dependencies do not interfere with other Python packages you may have installed. Here we choose `cuqipy` as the environment's name and also install pip in it with the following command
```